object-browser.md

```diff
@@ -0,0 +1,55 @@
+```hbs
+{{frost-object-browser
+  items=<sorted-filtered-selected-expanded-items>
+  header=(component ‘frost-info-bar’
+    icon=(hash
+      name=‘foo’
+      pack=‘foo’
+    )
+    title=‘foo’
+    description=‘foo’ or (component ‘foo’ …
+    scope=(component ‘foo’ ...
+    buttons=(array
+      (hash
+        icon=(hash
+          name=‘foo’
+          pack=‘foo’
+        )
+        label=‘foo’
+        onClick=(action ‘foo’ …
+      )
+    )
+  )
+  facets=(component ‘frost-bunsen-form’ / ‘frost-bunsen-facets’ …
```

Comment:
Sorry I don't have a lot of background with the object-browser. What is this facets?